Python: Getting Started bottle (#7824)
getsentry/sentry-docs: src/platforms/python/guides/bottle/index.mdx

Install `sentry-sdk` from PyPI with the `bottle` extra:

```bash
pip install --upgrade 'sentry-sdk[bottle]'
```

## Configure

> If you have the `bottle` package in your dependencies, the Bottle integration will be enabled automatically. There is nothing to do for you except initializing the Sentry SDK.

Reviewer: ```suggestion
If you have the `bottle` package in your dependencies, the Bottle integration will be enabled automatically when you initialize the Sentry SDK.
```

<SignInNote />

```python
from bottle import Bottle
import sentry_sdk

sentry_sdk.init(
    dsn="___PUBLIC_DSN___",
    # Set traces_sample_rate to 1.0 to capture 100%
    # of transactions for performance monitoring.
    traces_sample_rate=1.0,
)

app = Bottle()
```

## Verify

```python
from bottle import Bottle, run

sentry_sdk.init(...)  # same as above

app = Bottle()

@app.route('/')
def hello():
    1/0
    return "Hello World!"

run(app, host='localhost', port=8000)
```

> When you point your browser to [http://localhost:8000/](http://localhost:8000/) a transaction in the Performance section of [sentry.io](https://sentry.io) will be created. Additionally an error event will be sent to [sentry.io](https://sentry.io) and will be connected to the transaction.

Reviewer: ```suggestion
When you point your browser to [http://localhost:8000/](http://localhost:8000/) a transaction in the Performance section of [sentry.io](https://sentry.io) will be created. Additionally, an error event will be sent to [sentry.io](https://sentry.io) and will be connected to the transaction.
```

## Options

> By adding `BottleIntegration` explicitly to your `sentry_sdk.init()` call you can set options for `BottleIntegration` to change its behavior:

Reviewer: ```suggestion
If you add `BottleIntegration` explicitly to your `sentry_sdk.init()` call you can set options for `BottleIntegration` to change its behavior:
```
